commit cd1dc75d09e5ac8f5e55100e5b9099c85a91db7f
Author: Natanael Copa <ncopa@alpinelinux.org>
Date:   Tue Jul 20 09:02:05 2010 +0000

    setup-acf: install openssl
    
    so we can generate the certificate

diff --git a/setup-acf.in b/setup-acf.in
index 5144a21..4f2afec 100644
--- a/setup-acf.in
+++ b/setup-acf.in
@@ -11,7 +11,7 @@ usage() {
 	exit 0;
 }
 
-pkgs="acf-core acf-alpine-baselayout acf-apk-tools"
+pkgs="acf-core acf-alpine-baselayout acf-apk-tools openssl"
 
 while getopts "ae:hl:n" opt ; do
 	case $opt in
